generators/app/tasks/copyTemplateFiles.js

```diff
@@ -30,6 +31,10 @@ module.exports = function copyTemplateFiles() {
     bindedCopy(FLOWCONFIG_PATH.src, FLOWCONFIG_PATH.destination);
   }
 
+  if (!this.features['seamless-immutable']) {
```

Comment:
I don't see an `else` for when it does have seamless immutable. I'm guessing this copies the files that depend on seamless-immutable and then, if seamless-immutable wasn't chosen, it replaces them with the files that use deepFreeze.

If that's the case, could we avoid copying the same files twice?

Reply:
hmm i cant do that because we are copying entire folders, and just some files change when seamless-immutable is selected, i'd have to specify every file that is being copied if i want to skip some, i think i'ts better this way, what do you think?